### PR TITLE
[WIP] converted list() to []

### PR DESCRIPTION
**Pull request by auvipy**
clean ups

### CODE REVIEW

**Comment by auvipy** [comment]
need help to fix the bandit issue :/

**Comment by thedrow** [comment]
I fixed the bandit errors.

File: t/integration/test_canvas.py [hunk 1]
```diff
@@ -89,10 +89,10 @@ def test_group_chord_group_chain(self, manager):
 
         result = (before | connect | after).delay()
         result.get(timeout=TIMEOUT)
-        redis_messages = list(map(
+        redis_messages = [map(
```
Comment (thedrow) [inline t/integration/test_canvas.py]: This change is invalid.
The purpose of the code is to turn a `map()` object into a list.
The code here is completely different. It changes the value of `redis_messages` into `[map()]` from `[b'<string converted to bytes if python2>', b'<another string converted to bytes if python2>']`.

File: t/integration/test_canvas.py [hunk 2]
```diff
@@ -114,10 +114,10 @@ def test_second_order_replace(self, manager):
 
         result = second_order_replace1.delay()
         result.get(timeout=TIMEOUT)
-        redis_messages = list(map(
+        redis_messages = [map(
```
Comment (thedrow) [inline t/integration/test_canvas.py]: Ditto

**Pull request closed without merging**